server/models/Message.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+// Create Schema for Users
+const MessageSchema = new Schema({
+    conversation: {
+        type: Schema.Types.ObjectId,
+        ref: 'conversations',
+    },
+    to: {
+        type: Schema.Types.ObjectId,
+        ref: 'user',
+    },
+    from: {
+        type: Schema.Types.ObjectId,
+        ref: 'user',
+    },
+    body: {
```

Comment:
Just a suggestion, maybe rename this to `content`, `req.body.body` in the controller above works but may look a bit confusing :)

Reply:
Another good idea. This is done